app/views/api_page/_get_forecast_weather.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class='weather-widget'>
+  <% params[:forecast] ? get_weather(params[:forecast],2) : get_weather(2) %>
```

Comment:
As we discussed earlier, this should go into POROS, models or services. The controller should be using these new objects to get the information you need in the view.